File: test/fixtures/output/javascript/jquery/query-encoded.js
const settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://mockbin.com/har?startTime=2019-06-13T19%3A08%3A25.455Z&endTime=2015-09-15T14%3A00%3A12-04%3A00",
  "method": "GET",
  "headers": {}
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/swift/urlsession/fixtures/query-encoded.swift
import Foundation

let url = URL(string: "https://httpbin.org/anything")!
var components = URLComponents(url: url, resolvingAgainstBaseURL: true)!
let queryItems: [URLQueryItem] = [
  URLQueryItem(name: "startTime", value: "2019-06-13T19%3A08%3A25.455Z"),
  URLQueryItem(name: "endTime", value: "2015-09-15T14%3A00%3A12-04%3A00"),
]
components.queryItems = components.queryItems.map { $0 + queryItems } ?? queryItems

var request = URLRequest(url: components.url!)
request.httpMethod = "GET"
request.timeoutInterval = 10

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))